### PR TITLE
Feat/short headline for related

### PR DESCRIPTION
**Pull request by aronshamash**
As part of https://nidigitalsolutions.jira.com/browse/REPLAT-2892. 

- For Related Articles only use `shortHeadline` with a fallback to `headline`

### CODE REVIEW

File: packages/related-articles/__tests__/web/__snapshots__/la2-with-style.web.test.js.snap [hunk 1]
```diff
@@ -408,7 +408,7 @@ exports[`default styles 1`] = `
                 <h3
                   className="leadHeadlineClass S5"
                 >
-                  Test Headline
+                  Defence of the Realm
```
Comment (craigbilner) [inline packages/related-articles/__tests__/web/__snapshots__/la2-with-style.web.test.js.snap]: should this have changed?

File: packages/related-articles/__tests__/android/__snapshots__/la2.android.test.js.snap [hunk 2]
```diff
@@ -193,7 +193,7 @@ exports[`5. one lead and one support related article 1`] = `
                 accessibilityRole="heading"
                 aria-level="3"
               >
-                Second Headline
+                Size matters, say very tiny ministers
```
Comment (colincclark) [inline packages/related-articles/__tests__/android/__snapshots__/la2.android.test.js.snap]: better to stick with the test headline

**Comment by aronshamash** [comment]
@craigbilner @colincclark snapshots updated now. Can you have another look.

**Comment by craigbilner** [comment]
mega rebase

**Pull request closed without merging**